App/Sagas/ImageSharingSagas.ts

```diff
@@ -14,14 +14,14 @@ import {ActionType} from 'typesafe-actions'
 export function * shareWalletImage (id: string, threadId: string, comment?: string) {
   try {
     // TODO: Insert some state into the processing photos redux in case this takes long or fails
-    const blockId: string = yield call(TextileNode.sharePhotoToThread, id, threadId, comment)
+    const blockId: TT.BlockId = yield call(TextileNode.sharePhotoToThread, id, threadId, comment)
     yield put(TextileNodeActions.getPhotoHashesRequest(threadId))
```

Comment:
There are still places like this where you're passing a `string` (`threadId`) into something typed with the enum (`getPhotoHashesRequest`). Do we want to go though and make sure we take the typing as far as we can? I'm guessing the string -> enum un-typed scary conversion could be isolated to API responses and input from the user (the UI).

Reply:
Looks like other similar cases are typed, so maybe I just happened to look at the one you forgot...

Reply:
yeah, shouldn't ever be typed as string anymore